client/web/compose/src/components/Public/Record/BulkEdit/index.vue

```diff
@@ -179,6 +184,10 @@ export default {
     getFieldSelectorPlaceholder () {
       return this.$t(`recordList.bulkRecord.field.add${this.fields.length ? 'Another' : ''}`)
     },
+
+    selectedQuery () {
+      return this.filterQuery + this.selectedRecords.map(id => `recordID=${id}`).join(' OR ')
+    },
   },
 
   watch: {
```

Comment:
Replace selectedRecords watcher with query

```
    query: {
      handler (query) {
        if (!this.openOnSelect || !query.length) return

        this.record = new compose.Record(this.module, this.initialRecord)
        this.showModal = true
      },
    },
```

and add `this.$emit('close')` to the end of onModalHide

```
    onModalHide () {
      this.showModal = false

      if (this.openOnSelect) {
        this.fields = []
        this.record = new compose.Record(this.module, {})
      }
      this.$emit('close')
    },
```